spec/indico_batch_spec.rb

```diff
@@ -21,6 +21,15 @@
     expect(Set.new(response[1].keys)).to eql(expected_keys)
   end
 
+  it 'should tag text with correct emotion tags' do
```

Comment:
However in general I feel weird having this being the responsibility of the client libs